### ISSUE TITLE
Make file processing options object optional

### ISSUE THREAD

**Issue by bahmutov** [abramenal/cypress-file-upload#94]
Using `cypress-file-upload@3.3.2`

If my test is

```js
    cy.get('#file1').upload({
      fileContent: 'file contest',
      fileName: 'test.txt',
      mimeType: 'plain/text',
    })
```

then the plugin crashes trying to access `subjectType` of the undefined second object

<img width="1276" alt="Screen Shot 2019-08-01 at 10 25 11 AM" src="https://user-images.githubusercontent.com/2212006/62301573-dec9fa80-b446-11e9-8164-27e4dc5bba9f.png">

Adding the default subject type `input` fixes it, but should not even be required

```js
    cy.get('#file1').upload(
      {
        fileContent: 'file contest',
        fileName: 'test.txt',
        mimeType: 'plain/text',
      },
      { subjectType: 'input' }
    )
```

**Comment by bahmutov** [comment]
doing this for https://github.com/cypress-io/cypress-example-recipes/pull/319

**Comment by abramenal** [comment]
Hey @bahmutov 
Thanks for submitting the issue, I'm happy that Cypress core team pays attention to community contributions!

That's a nice one, for some reason never thought about that though it is quite useful.
Fixed with v3.3.3.

@all-contributors add @bahmutov for ideas
